doc/user/content/get-started/install-materialize-emulator.md

```diff
@@ -118,6 +118,26 @@ Community on Slack](https://materialize.com/s/chat).
 - By downloading the Docker image, you are agreeing to Materialize's
   [privacy policy](https://materialize.com/privacy-policy/).
 
+#### Materialize Self-Managed Community Edition or the Materialize Emulator Privacy FAQ
```

Comment:
https://preview.materialize.com/materialize/31800/get-started/install-materialize-emulator/#license-and-privacy-policy